geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/ConvertCommand.scala

```diff
@@ -38,7 +38,7 @@ class ConvertCommand extends Command {
     import scala.collection.JavaConversions._
 
     val sft = CLArgResolver.getSft(params.spec)
-    logger.info(s"Using SFT definition: ${sft}")
+    logger.info(s"Using SFT definition: $sft")
```

Comment:
Why does this use neither the Command.user logger nor the Command.output logger?